docs/howto/logging-to-a-file/index.adoc

```diff
@@ -0,0 +1,82 @@
+
+[#logging-to-a-file_{context}]
+= Logging to a file
+
+In addition to the console logging, you can save the logs of your application in a file.
+Typically, deployments use rotating logs to save disk space.
+
+In WildFly Swarm, logging is configured using system properties.
+Even though it is possible to use the `-Dproperty=value` syntax when launching your application, it is strongly recommended to configure file logging using the YAML profile files.
+
+.Prerequisites
+
+* A Maven-based application with the logging fraction enabled. For more information, see xref:enabling-logging_{context}[].
+* A writable directory on your file system.
+
+.Procedure
+
+. Open a YAML profile file of your choice.
+If you do not know which one to use, open `project-defaults.yml` in the root directory of the WAR archive with your application.
```

Comment:
I don't think we'd recommend modifying WAR files directly. Typically, in a Maven project, these config files are located in `src/main/resources`.